src/module.h

```diff
@@ -21,6 +22,12 @@ extern RedisModuleCtx *RSDummyContext;
 /** Indicates that RediSearch_Init was called */
 extern int RS_Initialized;
 
+#define RS_AutoMemory(ctx)                      \
```

Comment:
Why you need this then?

Reply:
If its only for legacy gc lets drop legacy gc as well

Reply:
It is for the coordinator.